src/components/Search/Profiles.tsx

```diff
@@ -29,10 +29,8 @@ const Profiles: FC<Props> = ({ query }) => {
     skip: !query
   });
 
-  // @ts-ignore
-  const profiles = data?.search?.items;
-  // @ts-ignore
-  const pageInfo = data?.search?.pageInfo;
+  const profiles = data?.search && (data.search as ProfileSearchResult)?.items;
+  const pageInfo = data?.search && (data.search as ProfileSearchResult)?.pageInfo;
```

Comment:
```suggestion
  const search = data?.search as ProfileSearchResult;
  const profiles = search?.items;
  const pageInfo = search?.pageInfo;
```

Can we do it as something like this? I feel this to be more readable 🤔

Reply:
Yes please. Make sure it doesn’t affect anything else.

Reply:
Thanks for the suggestion! It's now changed in all occurences. I think is much legible now 👍